sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -273,6 +281,12 @@ class SQLAppStatusListener(
       exec.metricsValues = aggregateMetrics(exec)
       exec.completionTime = Some(new Date(time))
       exec.endEvents += 1
+      // Update both physicalPlanDescription and planGraph while the description changed.
```

Comment:
All the changes in `SQLAppStatusListener` and `SQLExecution` in e29ad45 is to support metastore operation phase happened in `HiveTableScanExec.rawPartitions`. Different from other phases, `rawPartitions` was initialized in execution but the UI and plan description generated before execution. If we think it's not worth to change the execution end event, the changes and metrics for rawPartitions will revert together.